vNext/src/Orleans/Shims/ClassesForTesting/CustomSerializerTestClasses.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using Orleans.CodeGeneration;
+using Orleans.Serialization;
+
+namespace UnitTests.CustomSerializerTestClasses
```

Comment:
Let's look at an alternative tomorrow, basically by modifying the App domain shim itself